ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -276,6 +276,24 @@ To retain the old configuration with no TCAM profile the default platform settin
 +      non_mlag: 1020
 ```
 
+### Default value for OSPF area has been set to `0.0.0.0`
+
```

Comment:
We may not need to mention this in porting guide.

Reply:
Yes, we should include details on how to revert the default. I will take care of it.